buildstockbatch/gcp/gcp.py

```diff
@@ -379,21 +559,165 @@ def run_batch(self):
         # Start the job!
         created_job = client.create_job(create_request)
 
-        logger.debug(created_job)
         logger.info('Newly created GCP Batch job')
-        logger.info(f'Job name: {created_job.name}')
-        logger.info(f'Job UID: {created_job.uid}')
+        logger.info(f'  Job name: {created_job.name}')
+        logger.info(f'  Job UID: {created_job.uid}')
 
     @classmethod
-    def run_job(self, job_id, gcs_bucket, gcs_prefix, job_name, region):
+    def run_task(cls, task_index, job_name, gcs_prefix):
```

Comment:
FYI, I didn't look at most of this code block closely since it's mostly identical to the existing aws code.